bite-size-notes/week-9-e-recursive-bayesian-filter-gaussian.md

```diff
@@ -0,0 +1,212 @@
+# Recursive Bayseian Filters for Linear Gaussian Systems
+
+### Scope
+
+The recursive Bayesian filter describes how the probability distribution of a random signal varies with time.
+Astoundingly, if the signal is the output of a linear system with a Gaussian state distribution, the recursive Bayesian filter exactly reproduces the Kalman filter.
+
+### Objective
+
+Derive the Kalman filter from the recursive Bayesian filter subject to linear dynamics and Gaussian distributions, and comment on the connection between the two filters.
+
+## Introduction
+
+The recursive Bayesian filter (RBF) is the most general possible statement of how one's belief about the value of a random signal changes over time.
+If one wishes to estimate the state of an uncertain system, the RBF exactly describes how the probability of each state varies over time.
+This is the math that underlies any state estimation problem, from localizing the position of an aircraft using GPS to determining the orientation of a satellite using a star tracker.
+
+However, the RBF is challenging to use in practice because of the difficulty of computing the normalization factor in Bayes' rule, upon which the RBF is based, for systems with many possible states.
```

Comment:
Can you elaborate on this part? Why is it difficult to compute normalization factor in Bayes' rule?

Reply:
Ooh yeah, that's something I should have covered. I've added a paragraph explaining the problem!